codecs/codecs-btp/src/main/java/org/interledger/codecs/btp/framework/BtpCodecContextFactory.java

```diff
@@ -1,4 +1,4 @@
-package org.interledger.btp.asn.framework;
+package org.interledger.codecs.btp.framework;
```

Comment:
With the new structure introduced by this PR, I think this package should just be `org.interledger.codecs.btp`. The usage of the extra package named `framework` in each module just for a single class doesn't seem to add any value.

```suggestion
package org.interledger.codecs.btp;
```

Reply:
LGTM!